src/app.js

```diff
@@ -1,21 +1,35 @@
+/* eslint-disable no-console */
 'use strict';
 
-/**
- * Implement sum function:
- *
- * Function takes 2 numbers and returns their sum
- *
- * sum(1, 2) === 3
- * sum(1, 11) === 12
- *
- * @param {number} a
- * @param {number} b
- *
- * @return {number}
- */
-function sum(a, b) {
-  // write code here
-  return a + b;
-}
-
-module.exports = sum;
+const http = require('http');
+const fs = require('fs');
+const url = require('url');
+
+const PORT = process.env.PORT || 3000;
+
+const server = http.createServer((req, res) => {
+  const normalizedURL = new url.URL(req.url, `http://${req.headers.host}`);
+
+  if (!normalizedURL.pathname.startsWith('/file/')) {
+    res.statusCode = 400;
+    res.end('Pathname should start from "/file/"');
```

Comment:
can be less
```suggestion
    res.status(400).end('Pathname should start from "/file/"');
```
check other places